entrypoint.sh

```diff
@@ -1123,20 +1145,20 @@ function cleanup_driver_inventory() {
             continue
         fi
 
-        driver_ver_dirs=$(ls ${NVIDIA_NIC_DRIVERS_INVENTORY_PATH}/${kernel_ver_dir})
+        driver_ver_items=$(ls ${NVIDIA_NIC_DRIVERS_INVENTORY_PATH}/${kernel_ver_dir})
 
-        found_driver_ver_dirs=0
-        removed_driver_ver_dirs=0
-        for driver_ver_dir in ${driver_ver_dirs}; do
-            found_driver_ver_dirs=$((${found_driver_ver_dirs}+1))
+        found_driver_ver_items=0
+        removed_driver_ver_items=0
+        for driver_ver_item in ${driver_ver_items}; do
+            found_driver_ver_items=$((${found_driver_ver_items}+1))
 
-            if [ ${driver_ver_dir} != ${NVIDIA_NIC_DRIVER_VER} ]; then
-                exec_cmd "rm -rf ${NVIDIA_NIC_DRIVERS_INVENTORY_PATH}/${kernel_ver_dir}/${driver_ver_dir}"
-                removed_driver_ver_dirs=$((${removed_driver_ver_dirs}+1))
+            if [[ ${driver_ver_item} != "${NVIDIA_NIC_DRIVER_VER}" && ${driver_ver_item} != *.checksum ]]; then
```

Comment:
can we be explicit here and check for ${NVIDIA_NIC_DRIVER_VER}.checksum ?

Reply:
Yes, fixed that